recipe/meta.yaml

```diff
@@ -21,10 +21,10 @@ build:
 
 requirements:
   build:
-    - python
+    - python >=3.6
```

Comment:
Is python 3.6 needed? or is 3.5 enough?

Reply:
While it may work on Python 3.5 our infrastructure dropped Python 3.5 so I'm not confident to pin a version that conda-forge does not build/test. Let's keep 3.6 as the minimum here.